packages/oid4vci-demo-frontend/src/utils/mapper/branding/OIDC4VCIBrandingMapper.ts

```diff
@@ -51,26 +55,32 @@ export const getCredentialBrandings = async (metadata: EndpointMetadataResult):
     const credentialBranding = new Map<string, Array<IBasicCredentialLocaleBranding>>()
 
     Promise.all(
-        (metadata.credentialIssuerMetadata!.credentials_supported as CredentialSupported[]).map(async (credentialsSupported: CredentialSupported): Promise<void> => {
-            const localeBranding: Array<IBasicCredentialLocaleBranding> = await Promise.all(
-                (credentialsSupported.display ?? []).map(
-                    async (display: CredentialsSupportedDisplay): Promise<IBasicCredentialLocaleBranding> =>
-                        await credentialLocaleBrandingFrom(display)
-                ),
-            );
+        (Object.values(metadata.credentialIssuerMetadata!.credential_configurations_supported as Record<string, CredentialConfigurationSupported>))
+            .map(async (credentialsConfigSupported: CredentialConfigurationSupported): Promise<void> => {
+                const localeBranding: Array<IBasicCredentialLocaleBranding> = await Promise.all(
+                    (credentialsConfigSupported.display ?? []).map(
+                        async (display: CredentialsSupportedDisplay): Promise<IBasicCredentialLocaleBranding> =>
+                            await credentialLocaleBrandingFrom(display)
+                    ),
+                );
 
-            const types = 'types' in credentialsSupported ? credentialsSupported.types : undefined
-            if (types) {
-                const credentialTypes: Array<string> =
-                    types.length > 1
-                        ? types.filter((type: string) => type !== 'VerifiableCredential')
-                        : types.length === 0
-                            ? ['VerifiableCredential']
-                            : types
+                const types = 'types' in credentialsConfigSupported // TODO credentialsConfigSupported.types is deprecated
+                    ? credentialsConfigSupported.types as string[]
```

Comment:
Remove the comment or add a link to the ticket meant to handle it

Reply:
added ticket, we do not have a project for this so it is CWALL

Reply:
Please add a link to the ticket in the comment